src/xcode/ENA/ENA/Source/Scenes/Home/HomeViewController.swift

```diff
@@ -19,10 +19,18 @@ import ExposureNotification
 import UIKit
 
 protocol HomeViewControllerDelegate: AnyObject {
-	func homeViewControllerUserDidRequestReset(_ controller: HomeViewController)
+	func showRiskLegend()
+	func showExposureNotificationSetting(enState: ENStateHandler.State)
+	func showExposureDetection(state: HomeInteractor.State, isRequestRiskRunning: Bool)
+	func setExposureDetectionState(state: HomeInteractor.State, isRequestRiskRunning: Bool)
+	func showExposureSubmission(with result: TestResult?)
+	func showInviteFriends()
+	func showWebPage(from viewController: UIViewController, urlString: String)
+	func showAppInformation()
+	func showSettings(enState: ENStateHandler.State)
+	func addToUpdatingSetIfNeeded(_ anyObject: AnyObject?)
```

Comment:
Please add a comment what this method is doing, it's very generic and not understandable.

Reply:
Agree on this. I'm not a big fan of comments, as IMHO if something needs commenting then it's either too complex or too badly designed. I will come up with a better name for this instead.

Reply:
Renamed [here](https://github.com/corona-warn-app/cwa-app-ios/pull/834/commits/5dcc8f8b5dd25b92193c97f15ed0bdb8ee3de90c).